modules/manage/pages/manage-xdcr/replicate-using-scopes-and-collections.adoc

```diff
@@ -7,16 +7,16 @@ XDCR can be performed with reference to scopes and collections within source and
 == Understanding Scopes, Collections, and Replication
 
 As described in xref:learn:data/scopes-and-collections.adoc[Scopes and Collections], scopes and collections are containers for data within a bucket.
-A bucket can contain up to 100 scopes; and a scope can contain up to 1000 buckets.
+A cluster can contain up to 1000 scopes; and it can contain 1000 collections.
 
 As well as specifying a source bucket and a target bucket, XDCR can specify scopes and collections within those buckets.
-This allows data to be reassigned from one collection to another.
-It also provides a convenient way of _migrating_ data that currently resides in legacy buckets that do not support (or have not yet been established with) administrator-defined scopes and collections, into buckets, on a Version 7.0+ Couchbase-Server cluster, that _do_ contain administrator-defined scopes and collections.
+This allows data to be replicated from one collection to another.
 For more information, see xref:learn:clusters-and-availability/xdcr-overview.adoc#xdcr-with-scopes-and-collections[XDCR with Scopes and Collections].
 
 [#examples-on-this-page]
 == Examples on this Page
-The examples provided on this page &#8212; xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#migrate-data-to-a-collection[Migrate Data to a Collection] and xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#replicate-data-between-collections[Replicate Data Between Collections] &#8212; demonstrate how XDCR can be performed with reference to scopes and collections within source and target buckets, using Couchbase Web Console.
+
+The examples provided on this page &#8212; xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#migrate-data-to-a-collection[Migrate Data to a Collection], xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#replicate-data-between-collections-implicitly[Replicate Data Between Collections Implicitly], and xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc#replicate-data-between-collections-explicitly[Replicate Data Between Collections Explicitly] &#8212; demonstrate how XDCR can be performed with reference to scopes and collections within source and target buckets, using Couchbase Web Console.
 The examples assume the following:
 
 * A single-node cluster named `10.144.210.101` has been created, and features the `travel-sample` bucket.
```

Comment:
I'm sure you already know this but just wanted to reiterate that cluster names such as these may not be possible anymore due to https://issues.couchbase.com/browse/MB-39405

Reply:
Thanks. I will revise the examples to use the more appropriate cluster naming.